contracts/strategies/strategy-joe-farm-base.sol

```diff
@@ -107,4 +107,16 @@ abstract contract StrategyJoeFarmBase is StrategyBase {
             now.add(60)
         );
     }
+
+    function _takeFeeJoeToSnob(uint256 _keepJOE) internal {
+        IERC20(joe).safeApprove(joeRouter, 0);
+        IERC20(joe).safeApprove(joeRouter, _keepJOE);
+        path = new address[joe, wavax, snob];
+        _swapTraderJoeWithPath(path, _keepJOE);
+        uint _snob = IERC20(snob).balanceOf(address(this));
```

Comment:
`require(IERC20(snob).balanceOf(address(this)) > _keepJOE, "Inappropriate swap");
        IERC20(snob).safeTransfer(
            IController(controller).treasury(),
            _keepJOE`